src/app/static/src/scss/style.scss

```diff
@@ -162,3 +162,13 @@ a.down::after {
   text-align: left !important;
   min-width: 32em;
 }
+
+.noDataMessage {
```

Comment:
For some reason, the class from the stylesheet in the vue-chart repo isn't being applied properly in hint so I've added a copy here as well, which fixes the problem.

Reply:
You don't need a new css, there's a class you can use, see usage https://github.com/mrc-ide/hint/blob/5b4730fa002956a8da762a74cb3f5f1eb9b69e66/src/app/static/src/app/components/plots/MapEmptyFeature.vue#L2.

Reply:
I don't think it is possible to apply these classes as the no data message html lives inside the vue-charts repo. Ideally, the repo would provide its own css styling, but that doesn't seem to be getting applied whenever it is used within hint (although the repo's classes are applied when the example html in the repo are run directly in the browser). The vue-chart's stylesheet was suspiciously empty of classes prior to me adding my class to it, which suggests this might have been a known issue with the repo.

Reply:
See this pr for context: https://github.com/reside-ic/vue-charts/pull/30